jetstream/ordered.go

```diff
@@ -386,13 +387,19 @@ func (c *orderedConsumer) reset() error {
 	defer c.Unlock()
 	defer atomic.StoreUint32(&c.resetInProgress, 0)
 	if c.currentConsumer != nil {
+		c.currentConsumer.Lock()
+		if c.currentConsumer.subscriptions[""] != nil {
+			c.currentConsumer.subscriptions[""].Stop()
+		}
+		consName := c.currentConsumer.CachedInfo().Name
```

Comment:
are we confident that consumer name is updated after recreation, so we always delete the proper one?

Reply:
Should be good, in line 443 we do:
```go
c.currentConsumer = cons.(*pullConsumer)
```

This is done right after successful create.